### PR TITLE
Remove contracts autodeploy from repo

### LINKED ISSUE #2811: Move contract deployment/update code from this repository
[Related issue](https://github.com/nspcc-dev/neofs-contract/issues/394)

### PR DESCRIPTION
**Pull request by smallhive**
Closes #2811.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Changelog for NeoFS Node
 - Supporting of `__NEOFS__NETMAP*` X-headers (#2751)
 - Option to use insecure TLS cipher suites (#2755)
 - Counter metrics that were deprecated since v0.38.0 (#2798)
+- IR contracts deployment code. Moved to the contracts repo (#2812)
```
Comment (roman-khimov) [inline CHANGELOG.md:35]: We've started a new cycle.

**Review by roman-khimov — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

File: pkg/util/glagolitsa/glagolitsa.go [hunk 2]
```diff
@@ -54,3 +54,13 @@ const Size = 41
 func LetterByIndex(ind int) string {
 	return script[ind]
 }
+
+// Glagolitsa implement [github.com/nspcc-dev/neofs-contract/deploy.Glagolitsa].
```
Comment (cthulhu-rider) [inline pkg/util/glagolitsa/glagolitsa.go:58]:
```suggestion
// Glagolitsa implements [github.com/nspcc-dev/neofs-contract/deploy.Glagolitsa].
```
btw interface name degenerates the abstraction, there is only one glagolitsa. It'd be more clear with `Glagolitsa implements Alphabet`. We cant change it here unfortunately

File: pkg/innerring/innerring.go [hunk 3]
```diff
@@ -574,6 +576,8 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
 		if err != nil {
 			return nil, fmt.Errorf("init Sidechain witness scope: %w", err)
 		}
+
+		server.log.Info("autodeploy completed")
```
Comment (cthulhu-rider) [inline pkg/innerring/innerring.go:580]: this behavior change isnt related to the refactor topic so deserves separate commit

**Review by cthulhu-rider — APPROVED** [review]
